CHANGELOG.rst

```diff
@@ -7,6 +7,25 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 2.2.0 (2024-11-18)
+--------------------------
+
+* Added: ``audbackend.checksum()``
+  for calculating MD5 sum of files.
+  For parquet files
+  it considers the ``"hash"`` metadata entry instead,
+  if it is present
+* Changed: retry to re-establish connection to backend two times,
+  before finally failing
+* Removed: deprecated functions
+  ``audbackend.access()``,
```

Comment:
Is there a reason why the deprecated functions are on a newline each? Was the intention to have this rendered when the CHANGELOG file is converted by sphinx (and should it become a bulleted list) or is it just meant to be in the rest?